src/applications/gi/containers/VetTecSearchPage.jsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import { connect } from 'react-redux';
 import { withRouter } from 'react-router';
 import Scroll from 'react-scroll';
-import _ from 'lodash';
+import { isEqual, pick, omit } from 'lodash';
```

Comment:
^^ https://github.com/department-of-veterans-affairs/vets-website/pull/11673/files#r377317714